frontend/src/components/associations/marketplace/counter/CounterOrderMaker.tsx

```diff
@@ -3,7 +3,7 @@ import React, { useContext, useState } from "react";
 import { ToastContext, ToastLevel } from "../../../utils/Toast";
```

Comment:
![image](https://user-images.githubusercontent.com/11032610/83331841-9b276380-a298-11ea-9d58-8e3827fa3e30.png)


The arrow is a bit blown up here, and does not look like the other arrows used throughout the rest of the website. But I am not against this one specifically of course, we should just choose one version.

The arrow and the button « Recréditer le compte » are not aligned with the central card